pandas/io/formats/format.py

```diff
@@ -937,13 +943,18 @@ def to_latex(
     def _format_col(self, i: int) -> List[str]:
         frame = self.tr_frame
         formatter = self._get_formatter(i)
+        if self.index:
+            leading_space = "compat"
```

Comment:
Seems like there was already some discussion around this here https://github.com/pandas-dev/pandas/pull/25000/files#r252237505 -

Reply:
emm, i think i addressed it already in this PR?